### PR TITLE
Add support for python3.7

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: htmltreediff/util.py [hunk 1]
```diff
@@ -394,12 +409,24 @@ def tree_text(node):
     return ' '.join(text)
 
 
+# I really don't like that this seems to be required. But the old way doesn't
+# work (we were setting a read-only property). I am not certain if this will
+# work in all cases, but running against more HTML blobs will certainly help
+# determine that. And that will happen soon.
+@contextmanager
+def _patch_recursion_limit():
```
Comment (jlward) [inline htmltreediff/util.py]: This is basically the only sketchy thing in here. Not really sure what else to do.

Comment (sbrand83) [reply htmltreediff/util.py]: `minidom` is the thing that doesn't work anymore?

Comment (sbrand83) [reply htmltreediff/util.py]: maybe you could convert the passed in dom to xml and then parse it back into another dom object?

**Comment by sbrand83** [comment]
## Reviewer Checklist

### Required Items

- [ ] Beta Test Plan is posted and plan sufficiently exercises new/modified functionality from a user's perspective

- [ ] I checked boxes on all Pre-Review Checklist items indicating my agreement with that item's categorization as either "Non-applicable" or "Required".

- [ ] Pull request is spec'd and implementation matches spec

- [ ] Code complies with quality standards

### Not Applicable

- [ ] Code is unit tested (including performance tests) with appropriate coverage

- [ ] Docs are updated

- [ ] Evaluated database performance for large datasets, including slow_query_log review

- [ ] JavaScript changes have appropriate Qunit and Selenium coverage

- [ ] UX/UI review completed

File: htmltreediff/util.py (hunk 1 above)
Comment (jlward) [reply htmltreediff/util.py]: That's a good idea, but it doesn't work. It seems to mangle a bunch of things.

**Pull request merged**